File: app/src/main/java/com/soutvoid/gamesproject/domain/character/Character.java
package com.soutvoid.gamesproject.domain.character;

import com.soutvoid.gamesproject.domain.Image;

import java.io.Serializable;
import java.util.ArrayList;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * Created by andrew on 22-Feb-17.
 */

@Data
@AllArgsConstructor
public class Character implements Serializable {

    private Integer id;
    private String name;
    private Long createdAt;
    private Long updatedAt;
    private String slug;
    private String url;
    private Image mugShot;
    private Integer gender;
    private ArrayList<String> akas;
    private Integer species;
    private ArrayList<Integer> people = null;
    private ArrayList<Integer> games = null;

}

File: app/src/main/java/com/soutvoid/gamesproject/domain/character/Character.kt
package com.soutvoid.gamesproject.domain.character

import com.soutvoid.gamesproject.domain.Image
import java.io.Serializable
import java.util.*


data class Character(
        var id: Int,
        var name: String,
        var createdAt: Long?,
        var updatedAt: Long?,
        var slug: String?,
        var url: String?,
        var mugShot: Image?,
        var gender: Int?,
        var akas: ArrayList<String>?,
        var species: Int?,
        var people: ArrayList<Int>,
        var games: ArrayList<Int>
) : Serializable
